src/lib/logger.ts

```diff
@@ -353,6 +353,7 @@ export class Logger {
       tzOffset: timeZoneOffsetString(),
       event,
       method,
+      activityUrl: user.activityUrl,
```

Comment:
I should have mentioned this earlier, but this should be moved up to line 342 here also (after `application`) and then it can be merged.